draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -529,17 +532,23 @@ struct {
     opaque right[hash.length];
 } HashNodeInput;
 
+struct {
+    SubjectType subject_type;
+    opaque subject_info_hash[hash.length];
+    Claim claims<0..2^16-1>;
+} AbridgedAssertion;
```

Comment:
Somewhere (security considerations? the Transparency section?) I think we should discuss the implications of abridged assertions. Something to the effect that it still allows domain owners to recognize unauthorized issuance. You check, for all abridged assertions with a claim that matches your domain, and if any hashes don't match one you authorized, report a misissuance even though the preimage is not available.

And probably we should mention that, unlike CT, folks cannot search the logs for weak keys, since we've hashed them down, unless the CA provides the unabridged values somewhere. (Though given the whole point is to reduce the serving load with PQ keys...)

Reply:
Continuing in [the other thread](https://github.com/davidben/merkle-tree-certs/pull/49#discussion_r1313834970).